docs/Developer-guide/Development-Workflow.md

```diff
@@ -344,16 +344,22 @@ code change is actually verified by the test case.
 Regression tests (i.e, execution of all test cases accumulated with
 every commit) is not automatically triggered as the test cases can be
 extensive and is quite expensive to execute for every change submission
-in the review/resubmit cycle. Instead it is triggered by the
-maintainers, after code review. Passing the regression test is a
+in the review/resubmit cycle. Instead it is triggered when the patch
+contributor issues a +1 on gerrit. Passing the regression test is a
 necessary condition for merge along with code review points.
 
-To run all regession tests locally, run below script from glusterfs root directory.
+To run all regression tests locally, run below script from glusterfs root directory.
 
 ```console
 # ./run-tests.sh
 ```
 
+To run a single regression test locally, run the below command.
+
+```console
+# prove <path_to_the_file>
```

Comment:
Maybe # prove -vf <path_to_the_file> as it provides detailed information?

Reply:
-vf being options I wasn't sure what are the suggested ones to be used. Like I'm used to using -r as well. though it isn't needed for a single test, it comes handy to use the same command in other situations as well. Anyways for now, I'll update for these two options alone. let me know if anything has to be added. Thanks @BarakSason